### PR TITLE
Remove more checkstyle suppressions

### PR DESCRIPTION
**Pull request by rjernst**
This commit removes several more line length violations from checkstyle
suppression.
relates #34884

### CODE REVIEW

**Comment by rjernst** [comment]
@elasticmachine test this please

File: server/src/main/java/org/elasticsearch/rest/action/cat/RestShardsAction.java [hunk 1]
```diff
@@ -170,14 +175,19 @@ protected Table getTableWithHeader(final RestRequest request) {
...
-        table.addCell("segments.index_writer_memory", "alias:siwm,segmentsIndexWriterMemory;default:false;text-align:right;desc:memory used by index writer");
-        table.addCell("segments.version_map_memory", "alias:svmm,segmentsVersionMapMemory;default:false;text-align:right;desc:memory used by version map");
-        table.addCell("segments.fixed_bitset_memory", "alias:sfbm,fixedBitsetMemory;default:false;text-align:right;desc:memory used by fixed bit sets for nested object field types and type filters for types referred in _parent fields");
+        table.addCell("segments.index_writer_memory",
+            "alias:siwm,segmentsIndexWriterMemory;default:false;text-align:right;desc:memory used by index writer");
+        table.addCell("segments.version_map_memory",
+            "alias:svmm,segmentsVersionMapMemory;default:false;text-align:right;desc:memory used by version map");
+        table.addCell("segments.fixed_bitset_memory",
+            "alias:sfbm,fixedBitsetMemory;default:false;text-align:right;desc:memory used by fixed bit sets for nested object" +
+                " field types and type filters for types referred in _parent fields");
```
Comment (matriv) [inline server/src/main/java/org/elasticsearch/rest/action/cat/RestShardsAction.java]: same here

File: server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java [hunk 2]
```diff
@@ -1509,7 +1517,8 @@ public void restore() throws IOException {
                     logger.trace("[{}] [{}] restoring from to an empty shard", shardId, snapshotId);
                     recoveryTargetMetadata = Store.MetadataSnapshot.EMPTY;
                 } catch (IOException e) {
-                    logger.warn(() -> new ParameterizedMessage("{} Can't read metadata from store, will not reuse any local file while restoring", shardId), e);
+                    logger.warn(() -> new ParameterizedMessage("{} Can't read metadata from store, will not reuse any local" +
+                        " file while restoring", shardId), e);
```
Comment (matriv) [inline server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java]: same here for the leading whitespace char.

File: server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java [hunk 3]
```diff
@@ -898,10 +900,12 @@ public void verify(String seed, DiscoveryNode localNode) {
...
                 } catch (IOException exp) {
-                    throw new RepositoryVerificationException(metadata.name(), "store location [" + blobStore() + "] is not accessible on the node [" + localNode + "]", exp);
+                    throw new RepositoryVerificationException(metadata.name(), "store location [" + blobStore() +
+                        "] is not accessible on the node [" + localNode + "]", exp);
                 }
             } else {
-                throw new RepositoryVerificationException(metadata.name(), "a file written by master to the store [" + blobStore() + "] cannot be accessed on the node [" + localNode + "]. "
+                throw new RepositoryVerificationException(metadata.name(), "a file written by master to the store [" + blobStore() +
+                    "] cannot be accessed on the node [" + localNode + "]. "
                     + "This might indicate that the store [" + blobStore() + "] is not shared between this node and the master node or "
```
Comment (matriv) [inline server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java]: maybe here and below you can move the `+` to the end of the prev line.

**Review by matriv — APPROVED** [review]
LGTM. Left some very minor comments regarding additional cosmetic changes.

**Comment by rjernst** [comment]
@elasticmachine run gradle tests 1

**Comment by rjernst** [comment]
@elasticmachine run gradle build tests 1

**Pull request merged**